src/main/cms/cms_menu_imu.c

```diff
@@ -108,15 +111,31 @@ static void setProfileIndexString(char *profileString, int profileIndex, char *p
     profileString[charIndex] = '\0';
 }
 
+static char pidProfileNames[PID_PROFILE_COUNT][MAX_PROFILE_NAME_LENGTH + PROFILE_INDEX_STRING_ADDITIONAL_SIZE];
+static const char* pidProfileNamePtrs[PID_PROFILE_COUNT];
```

Comment:
`static const char *pidProfileNamePtrs[PID_PROFILE_COUNT];`
see https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md

Reply:
Oh gosh, I hate working on multiple projects with different coding styles at the same time.
Isn't there a code styling check for Betaflight?..
Done.